src/containers/LoginForm.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Form, Button, Alert, Spinner } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import loginAction from '../store/actions/authActions/LoginAction';
+import validateLoginForm from '../utils/validation';
+import store from '../store/store';
+import { REGISTER, LOGIN_LOADING } from '../store/actions/actionTypes';
+
+export class LoginForm extends React.Component {
+  static propTypes = {
+    loginUser: PropTypes.object.isRequired,
```

Comment:
Do this to eliminate this error @MandelaK 
```suggestion
    loginUser: PropTypes.shape({}).isRequired,,
```